File: src/Entity/Performance.php
<?php

namespace App\Entity;

use App\Repository\PerformanceRepository;
use Doctrine\ORM\Mapping as ORM;

/**
 * @ORM\Entity(repositoryClass=PerformanceRepository::class)
 */
class Performance
{
    /**
     * @ORM\Id
     * @ORM\GeneratedValue
     * @ORM\Column(type="integer")
     */
    private $id;

    /**
     * @ORM\Column(type="string", length=255)
     */
    private $depart;

    /**
     * @ORM\Column(type="string", length=255)
     */
    private $arivee;

    /**
     * @ORM\Column(type="integer")
     */
    private $distance;

    /**
     * @ORM\Column(type="string", length=255)
     */
    private $temps;

    /**
     * @ORM\ManyToOne(targetEntity=utilisateur::class, inversedBy="performances")
     */
    private $id_util;

    public function getId(): ?int
    {
        return $this->id;
    }

    public function getDepart(): ?string
    {
        return $this->depart;
    }

    public function setDepart(string $depart): self
    {
        $this->depart = $depart;

        return $this;
    }

    public function getArivee(): ?string
    {
        return $this->arivee;
    }

    public function setArivee(string $arivee): self
    {
        $this->arivee = $arivee;

        return $this;
    }

    public function getDistance(): ?int
    {
        return $this->distance;
    }

    public function setDistance(int $distance): self
    {
        $this->distance = $distance;

        return $this;
    }

    public function getTemps(): ?string
    {
        return $this->temps;
    }

    public function setTemps(string $temps): self
    {
        $this->temps = $temps;

        return $this;
    }

    public function getIdUtil(): ?utilisateur
    {
        return $this->id_util;
    }

    public function setIdUtil(?utilisateur $id_util): self
    {
        $this->id_util = $id_util;

        return $this;
    }
}

File: E-bike_java/src/entities/performance.java
package entities;

/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */


import java.util.Objects;
import javafx.beans.property.SimpleIntegerProperty;
import javafx.beans.property.SimpleStringProperty;

/**
 *
 * @author user
 */
public class performance {
    
    private SimpleStringProperty depart;
    private SimpleStringProperty arivee;
    private SimpleStringProperty distance;
        private SimpleStringProperty temps;
            private SimpleStringProperty mail;
               private SimpleStringProperty role;
               SimpleIntegerProperty id;

    public performance() {
    }

    public performance(String depart, String arivee, String distance, String temps) {
        this.depart = new SimpleStringProperty(depart)  ;
        this.arivee = new SimpleStringProperty(arivee);
        this.distance = new SimpleStringProperty(distance);
        
        this.temps =new SimpleStringProperty(temps) ;
      
        
    }
   public performance(String depart, String arivee, String distance, String temps,int id) {
       this.depart = new SimpleStringProperty(depart)  ;
        this.arivee = new SimpleStringProperty(arivee);
        this.distance = new SimpleStringProperty(distance);
        
        this.temps =new SimpleStringProperty(temps) ;
                 this.id =new SimpleIntegerProperty(id) ;
        
    }
    
    

    public String getDepart() {
        return depart.get();
    }

    public void setDepart(String depart) {
        this.depart = new SimpleStringProperty(depart);
    }

    public String getArivee() {
        return arivee.get();
    }

    public void setArivee(String arivee) {
        this.arivee = new SimpleStringProperty(arivee);
    }
 public SimpleStringProperty getDepartProperty(){
        return depart;
    }
    public String getDistance() {
        return distance.get();
    }
   
    public SimpleStringProperty getDistanceProperty(){
        return distance;
    }
    public void setDistance(String distance) {
        this.distance = new SimpleStringProperty(distance);
    }
    public String getTemps() {
        return temps.get();
    }
   
    public SimpleStringProperty getTempsProperty(){
        return temps;
    }
    public void setTemps(String temps) {
        this.temps = new SimpleStringProperty(temps);
    }
   
   
    public SimpleStringProperty getAriveeProperty(){
        return arivee;
    }
     
          public int getId() {
        return id.get();
    }
   
    public SimpleIntegerProperty getIdProperty(){
        return id;
    }
    public void setId(int id) {
        this.id = new SimpleIntegerProperty(id);
    }
    @Override
    public int hashCode() {
        int hash = 3;
        return hash;
    }

 

  
}
